apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -215,95 +230,110 @@ export function Chat({
     const postIclResponse = (await new Promise((resolve) => {
       const timeout = setTimeout(() => {
         resolve({
           parsed: {},
-          question: userContent,
+          question: userContentRef,
           domain: chatbot?.categories[0].category.name as string,
           chatbot: chatbot?.name as string,
         });
         clearTimeout(timeout);
       }, 700);
     })) as { parsed: any; question: string; domain: string; chatbot: string };
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)